src/hooks.server.ts

```diff
@@ -13,7 +13,7 @@ export const handle: Handle = async ({ event, resolve }) => {
 		return resolve(event)
 
 	// If the app is prerendering during deploy, we don't want to run the rest of the code as it will cause env errors
-	if (prerendering) return await resolve(event)
+	if (building) return await resolve(event)
```

Comment:
Just wondering, what's the difference here?

Reply:
renamed by major bump